frontend/src/Pages/Home/Home.tsx

```diff
@@ -14,7 +14,7 @@ export default function Home({ navigation }: { route: any; navigation: any }) {
         if (!response.ok) {
           throw new Error(`${response.status}`);
         }
-        let json = await response.json();
+        let json = await response.text();
```

Comment:
Should be response.json()

Reply:
that gives me errors unfortunately, but i'll try